Blog post introducing 15 years celebrations (#1303)
ropensci/roweb3: content/blog/2026-05-20-anniversary/index.md

---
slug: anniversary2026
title: '15 years of rOpenSci, and we're just getting started 🎉'
author:

>     - rOpenSci Team

Author: ```suggestion
    - The rOpenSci Team
```

date: '2026-05-20'
tags:
  - rOpenSci Team
featured: true
description: 
params:

>   doi: 

Author: ```suggestion
  doi: 10.59350/cz2sy-c7z69
```

---
slug: anniversary2026

> title: "15 years of rOpenSci, and we're just getting started 🎉"

Reviewer: I love this title!
```suggestion
title: "15 Years of rOpenSci, and We're Just Getting Started 🎉"
```

author:
    - The rOpenSci Team
date: '2026-05-20'
tags:
  - rOpenSci Team
featured: true
description: 
socialImg: blog/2026/05/20/aniversary/PixelArt15yearrOpenSci.png
socialAlt: "Retro pixel-art graphic celebrating rOpenSci's 15th anniversary. The text 'rOpenSci' appears at the top in pixel font, flanked by three pixel-art balloons. A browser window frames the central message: '15 YEARS / TRANSFORMING OPEN SCIENCE' in bold pixel letters, overlaid on the rOpenSci geometric network pattern. A pixel badge reads 'OMG'. A pixel folder and sparkle icons complete the design."
social: "It started with an email in July 2011, and 15 years later, rOpenSci is throwing a Quinceañera! 🎂 We're marking the anniversary with a year of community activities, kicking off with virtual celebration events in June"
params:
  doi: 10.59350/cz2sy-c7z69
---

> Digging through our memory box, we came across a conversation in which we tried to piece together when it all began with rOpenSci. 

Reviewer: ```suggestion
Digging through our memory box, we came across a conversation from which we tried to piece together when it all began with rOpenSci. 
```

On July 13, 2011, an email was sent with the idea of a shared blog, a clever domain name, and a way to connect R package developers who cared about open science. The name "rOpenSci" appear in that email. A few months before that, the first commits had already been pushed to what would become taxize and treeBASE, two packages that quietly planted the seed of something much bigger.

That was 15 years ago. This year, we celebrate. 🎉

> {{< figure src = "PixelArt15yearrOpenSci.png" alt = "Retro pixel-art graphic celebrating rOpenSci's 15th anniversary. The text 'rOpenSci' appears at the top in pixel font, flanked by three pixel-art balloons. A browser window frames the central message: '15 YEARS / TRANSFORMING OPEN SCIENCE' in bold pixel letters, overlaid on the rOpenSci geometric network pattern. A pixel badge reads 'OMG'. A pixel folder and sparkle icons complete the design." >}}

Reviewer: Make it a tad smaller so you can see it all at once and then centred.
```suggestion
{{< figure src = "PixelArt15yearrOpenSci.png" alt = "Retro pixel-art graphic celebrating rOpenSci's 15th anniversary. The text 'rOpenSci' appears at the top in pixel font, flanked by three pixel-art balloons. A browser window frames the central message: '15 YEARS / TRANSFORMING OPEN SCIENCE' in bold pixel letters, overlaid on the rOpenSci geometric network pattern. A pixel badge reads 'OMG'. A pixel folder and sparkle icons complete the design." width = "500" class = "center">}}
```

## _Quinceañera_ time 🎂

> Fifteen years is a milestone worth marking properly, that is why we want to celebrate with our community. We have a full year of activities planned, and we want you along for all of it.

Reviewer: It'd be nice to explain what a Quinceañera is. A reasonable number of North Americans will understand, but I suspect many, if not most, Europeans/Africans/Asians might not be familiar. 

Perhaps something like this? Or does that sound too stuffy? Perhaps explain it in your voice?

```suggestion
Fifteen years[^1] is a milestone worth marking properly, that is why we want to celebrate with our community. We have a full year of activities planned, and we want you along for all of it.

[^1]: The fifteenth birthday of young women in many Latin American cultures is a special marker of adulthood called a Quinceañera.
```

> Expect several diverse events, retrospectives and a few surprises we're still stitching together. We'll be reflecting on what we've built, amplifying the work of contributors old and new, and dreaming out loud about the next 15 years.

Reviewer: ```suggestion
Expect several diverse events, retrospectives and a few surprises we're still stitching together. We'll be reflecting on what we've built, highlighting the work of contributors old and new, and dreaming out loud about the next 15 years.
```

Stay tuned to this blog and our newsletter for announcements as the year unfolds.

> ## First up: co-working session and virtual casual community celebration

Reviewer: I got a bit hung up on the first order, but it wasn't wrong, it just tweaked my brain funny.
```suggestion
## First up: co-working session and casual virtual community celebration
```

> We're kicking things off with one [co-working session on Tuesday, June xx]() and two virtual community events on [Wednesday, June 10]() and [Wednesday, June 17]() in different timezones, so as many people as possible can join.

Reviewer: Let's call the second two events "celebrations" so we're clear that we're not talking about coworking.
```suggestion
We're kicking things off with one [co-working session on Tuesday, June 2](/events/coworking-2026-06) and two virtual celebrations on [Wednesday, June 10](/events/celebrations-2026-06-10) and [Wednesday, June 17](/events/celebrations-2026-06-17) in different timezones, so as many people as possible can join.
```

> Each 90-minute session is built around rotating small-group conversations. You'll meet community members you may not know yet, and together you'll dig into questions to reflect on rOpenSci's past and future.

Reviewer: ```suggestion
Each 90-minute celebration is built around rotating small-group conversations. You'll meet community members you may not know yet, and together you'll dig into questions to reflect on rOpenSci's past and future.
```

social: "It started with an email in July 2011, and 15 years later, rOpenSci is throwing a Quinceañera! 🎂 We're marking the anniversary with a year of community activities, kicking off with virtual celebration events in June"
params:
  doi: 10.59350/cz2sy-c7z69
---

Digging through our memory box, we came across a conversation from which we tried to piece together when it all began with rOpenSci. 

On July 13, 2011, an email was sent with the idea of a shared blog, a clever domain name, and a way to connect R package developers who cared about open science. The name "rOpenSci" appear in that email. A few months before that, the first commits had already been pushed to what would become taxize and treeBASE, two packages that quietly planted the seed of something much bigger.

That was 15 years ago. This year, we celebrate. 🎉

> {{< figure src = "PixelArt15yearrOpenSci.png" alt = "Retro pixel-art graphic celebrating rOpenSci's 15th anniversary. The text 'rOpenSci' appears at the top in pixel font, flanked by three pixel-art balloons. A browser window frames the central message: '15 YEARS / TRANSFORMING OPEN SCIENCE' in bold pixel letters, overlaid on the rOpenSci geometric network pattern. A pixel badge reads 'OMG'. A pixel folder and sparkle icons complete the design." width = "500" class = "center">}}

Author: ```suggestion
{{< figure src = "PixelArt15yearrOpenSci.png" alt = "Retro pixel-art graphic celebrating rOpenSci's 15th anniversary. The text 'rOpenSci' appears at the top in pixel font, flanked by three pixel-art balloons. A browser window frames the central message: '15 YEARS / TRANSFORMING OPEN SCIENCE' in bold pixel letters, overlaid on the rOpenSci geometric network pattern. A pixel badge reads 'OMG'. A pixel folder and sparkle icons complete the design." width = "500" class = "center" caption = "Template design by Lauren Creatives in Canva. Adapted by Yani.">}}
```
